### PR TITLE
Add poster to the options the tech are receiving

### PR DESCRIPTION
**Pull request by eXon**
The only way I can know if I must set the default YouTube poster is by checking if the option poster is already set. However, that information is not sent to the tech right now.

### CODE REVIEW

**Comment by heff** [comment]
Makes sense to me. Does the video.js poster not cover up the youtube poster?

**Comment by eXon** [comment]
The tech needs to set the YouTube poster url. If we use the one displayed by the iframe, we have a big red ugly play button (really weird with the vjs poster). When we set a VJS poster, it hides the whole iframe until you click play.

File: src/js/player.js [hunk 1]
```diff
@@ -502,7 +502,8 @@ class Player extends Component {
       'autoplay': this.options_.autoplay,
       'preload': this.options_.preload,
       'loop': this.options_.loop,
-      'muted': this.options_.muted
+      'muted': this.options_.muted,
+      'poster': this.options_.poster
```
Comment (heff) [inline src/js/player.js]: @eXon could you change this to use `this.poster()`? That would make sure that if the poster changes, later techs will get the new poster.

**Comment by heff** [comment]
Merged in, thanks! I think we still have a little to figure out around the poster in #2339.

**Pull request closed without merging**